docs/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@xstate/docs",
+  "version": "1.0.0",
```

As this is not publishable to the registry we don't have to track the version. At the same time it's preferable to add a private flag so we don't accidentally publish this.
```suggestion
  "private": true,
```